### PR TITLE
Set useful CDN text selections on click (fixes #1326)

### LINKED ISSUE #1326: Change the mdl custom build cdn link to something easily copy-able?
The cutom build cdn link is much hard to copy as it need mouse drag,
double click selects more content than the needed link,

can you guys make it a copy-able text field or some thing that functions as seen on http://www.bootstrapcdn.com or as click to copy as on http://cdnjs.com/libraries/font-awesome ?

### PR DESCRIPTION
**Pull request by surma**
untitled-1

### CODE REVIEW

**Comment by surma** [comment]
Suprisingly, works across browsers :dancer:

**Comment by surma** [comment]
friendly ping about this

File: docs/_assets/customizer.js [hunk 1]
```diff
@@ -42,6 +42,33 @@ function init() {
     });
   });
 
+  var clickCtr = 0;
+  cdn.addEventListener('click', function() {
+    var selection = window.getSelection();
+    selection.removeAllRanges();
+
+    var range = document.createRange();
```
Comment (samccone) [inline docs/_assets/customizer.js]: i <3 range

**Comment by addyosmani** [comment]
Originally thought I had comments about the approach here, but on second looks this should be fine to land. Thanks @surma!

**Pull request merged**